docs/pages/docs/getting-started/app-router.mdx

```diff
@@ -14,7 +14,10 @@ If you haven't done so already, [create a Next.js app](https://nextjs.org/docs/g
 
 Let's get started!
 
-**Run `npm install next-intl` and create the following file structure:**
+**Run 
+```npm install next-intl```
+and create the following file structure:
+**
```

Comment:
This only breaks the formatting.

**Before:**

![Screenshot 2024-04-13 at 15 32 49](https://github.com/amannn/next-intl/assets/4038316/7af6b831-2436-4052-a19f-c09257e9ef14)

**After:**

![Screenshot 2024-04-13 at 15 32 31](https://github.com/amannn/next-intl/assets/4038316/16d68773-f40f-4c41-ae3a-e42c0b1e47e6)